recipes-ni/ni-test-boot-time/files/run-ptest

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+TEST_NAME="test_boot_time"
+TEST_LOG="/var/log/boottime"
+INFLUXDB_INFO="/home/admin/.influxdb.info"
+
+if [[ -f "$INFLUXDB_INFO" ]]; then
+	source "$INFLUXDB_INFO"
+	python3 upload_results.py "-i $TEST_LOG" "-s $INFLUXDB_SERVER" "-p $INFLUXDB_PORT"
+	RESULT=$?
+else
+	echo "INFO: InfluxDB connection information not found at '$INFLUXDB_INFO'"
+	RESULT=1
```

Comment:
For local ptest runs or runs of debug dashboard events (events with a "debug" tag), there will be no .influxdb.info file present on the target. I don't know if it's better for this test to "pass" in that case? This test is a little different from some other tests which have a logical pass/fail result independent from whether the data is uploaded or not.

Reply:
Since the main action of this test is uploading the boot time to influxdb for later comparisons I have decided it is better to fail the test if the upload fails. That way we would catch if something is wrong and we're not actually capturing results.

If you prefer, I could potentially "skip" the test in the cases where the influxdb info is not present. I am reluctant to always "pass" the test since it is going to hide cases were the reporting stopped working and we won't notice until much later when we see the data is missing.

Long term once we collect enough data from all devices we can consider adding device specific .conf files to this package with thresholds to test against and report a more accurate pass/fail. At this time we don't have enough data to establish these pass/fail thresholds.

Reply:
I went ahead and marked the test SKIP in this situation (see v3).